File: Packages/MeterKit/Sources/MeterCore/ReminderFrequency.swift
import Foundation

/// 用户自己打开的定时提醒频率。就这四档，不做自定义天数。
public enum ReminderFrequency: String, CaseIterable, Sendable, Codable, Equatable {
    case daily
    case weekly
    case biweekly
    case monthly
}

File: Android/app/src/main/kotlin/com/zhechengqi/tollcat/settings/ReminderFrequency.kt
package com.zhechengqi.tollcat.settings

object ReminderFrequency {
    const val DAILY = "daily"
    const val WEEKLY = "weekly"
    const val BIWEEKLY = "biweekly"
    const val MONTHLY = "monthly"

    val all = listOf(DAILY, WEEKLY, BIWEEKLY, MONTHLY)

    fun normalize(raw: String?): String {
        return all.firstOrNull { it == raw } ?: WEEKLY
    }
}
